### PR TITLE
[Form][TwigBridge] Add help_attr

### PR DESCRIPTION
**Pull request by mpiot**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no 
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Add help_attr to the  form_help method.

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/form_help.html.php [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <?php if (!empty($help)): ?>
-    <p id="<?php echo $view->escape($id); ?>_help" class="help-text"><?php echo $view->escape(false !== $translation_domain ? $view['translator']->trans($help, array(), $translation_domain) : $help); ?></p>
+    <?php $help_attr['class'] = trim((isset($help_attr['class']) ? $help_attr['class'].' help-text' : 'help-text')); ?>
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/form_help.html.php]: you could apply the trimming around the concatenation only, as the other alternative is already trimmed.

**Comment by xabbuh** [comment]
Is this still needed given that #26332 was merged?

**Comment by mpiot** [comment]
This follow the last, this PR add a 'help_attr', like 'label_attr' for exemple. That permit to configure the 'help' entry.

**Comment by xabbuh** [comment]
@mpiot LGTM, can you please rebase here?

**Comment by mpiot** [comment]
Mmm, I've broke something... Everything was okay on april, but test don't pass, and when I've try to rebase... apparently I've do something you didn't need... And my ability with Git is not really high...

**Comment by mpiot** [comment]
Mmm, sorry I don't really know what to do :-/
Change the:
    "require-dev": {
        ...
        "symfony/form": "^4.1",
        ...
    },
    "conflict": {
        ...
        "symfony/form": "^4.1",
        ...
    },

I replace all 4.1 per ... ? 4.2 ? But I need to declare somewhere the version change ?

And I need to do the same in the Twig Bridge ?

I'm so sorry, I don't really know how it works.

**Comment by mpiot** [comment]
@xabbuh Thanks

 I try to understand errors with deps=high. On my machine when I do something like:

    cd src/Symfony/Bundle/FrameworkBundle/
    composer update --no-progress --no-suggest --ansi
    phpunit --exclude-group tty,benchmark,intl-data

And tests are OK, I don't see the difference with Travis, because I use the same commands as in the travis test.

**Comment by xabbuh** [comment]
@mpiot This is a bit tricky. We use Travis to ensure that 4.1 versions of components work with 4.2 dependencies. The form theme tests are not very flexible as they mainly just compare the rendered HTML. I try to find a way to relax them a bit.

**Comment by mpiot** [comment]
@xabbuh thanks, so there's no way to pass the tests right now?

**Comment by xabbuh** [comment]
@mpiot I try to look into it tonight.

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
If I don't miss anything here, tests would pass just when we merge this into the `master` branch. We *could* skip the new tests in case the Form component is not available in a compatible version, but that would only be needed for this PR. Afterwards, FrameworkBundle as well as TwigBridge will conflict with these versions of the Form component. So IMO not worth it to me and I vote for just merging as is.

**Comment by mpiot** [comment]
Thank you for the time you spent watching this :-)

**Comment by fabpot** [comment]
Thank you @mpiot.

**Comment by xabbuh** [comment]
@mpiot FYI, @nicolas-grekas helped me to spot why the tests where failing (I move the test method from the Form component to the concrete subclasses in FrameworkBundle and TwigBridge to fix it).

**Comment by mpiot** [comment]
@xabbuh Thank you for this feedback, always important to know what was wrong :-)

**Pull request merged**